package.json

```diff
@@ -17,41 +17,47 @@
   "repository": "https://github.com/ahmadsoe/ember-highcharts.git",
   "scripts": {
     "build": "ember build",
-    "start": "ember server",
+    "lint:js": "eslint ./*.js addon addon-test-support app config lib server test-support tests",
+    "start": "ember serve",
     "test": "ember try:each"
   },
   "dependencies": {
-    "broccoli-funnel": "^1.0.1",
-    "broccoli-merge-trees": "^1.1.1",
-    "ember-cli-babel": "^5.1.7",
-    "ember-cli-htmlbars": "^1.1.1",
+    "broccoli-funnel": "^2.0.1",
+    "broccoli-merge-trees": "^1.2.0",
+    "ember-assign-polyfill": "^2.2.0",
+    "ember-cli-babel": "^6.6.0",
+    "ember-cli-htmlbars": "^2.0.1",
     "ember-getowner-polyfill": "^1.2.2"
```

Comment:
I believe this can be removed for Ember versions > 2.3

Reply:
This is an addon though, so we don't know what Ember version people will be using, so we need to keep it.